ui/components/ui/url-icon/url-icon.stories.js

```diff
@@ -1,10 +1,25 @@
 import React from 'react';
-import { text } from '@storybook/addon-knobs';
 import UrlIcon from './url-icon';
 
 export default {
   title: 'Components/UI/UrlIcon',
   id: __filename,
+  argType: {
+    name: { description: 'Icon Name', control: 'text' },
+    url: { description: 'Icon File URL', control: 'text' },
```

Comment:
Let's keep the control names the same as the prop names. I think that should help educated devs on the component api quicker.

```suggestion
    name: { control: 'text' },
    url: { control: 'text' },
```